### PR TITLE
Add support for more network scenarios

### PR DESCRIPTION
**Pull request by jtaleric**
Allow for :
- hostNetwork testing -- not using OpenShift SDN
- Pods to be scheduled to specific nodes using hostname

### CODE REVIEW

File: docs/uperf.md [hunk 1]
```diff
@@ -28,6 +33,22 @@ spec:
     runtime: 60
 ```
 
+`hostnetwork` will test the performance of the node the pod will run on.
+
+*Note:* If you want to run with hostnetwork, you will need to execute the following:
+
+```bash
+
+$ oc adm policy add-scc-to-user privileged -z benchmark-operator
```
Comment (aakarshg) [inline docs/uperf.md]: What's the way to do this through `kubectl` ? It'd be helpful to add that here and then put forward saying if you've oc installed then you can accomplish this through the above mentioned.

Comment (jtaleric) [reply docs/uperf.md]: Not sure. You can try this commit on a pure kube deployment to see if this is even needed.

Comment (jtaleric) [reply docs/uperf.md]: Added a note for OpenShift env. Unsure about Kube.

File: roles/uperf-bench/templates/workload_store.yml.j2 [hunk 2]
```diff
@@ -12,14 +12,19 @@ spec:
...
+{% endif %}
       containers:
       - name: benchmark
         image: "quay.io/benchmark-operator/uperf:latest"
         command: ["/bin/sh", "-c"]
         args:
           - "cat /tmp/uperf-test/uperfXML;
              export h={{item.status.podIP}};
-             uperf -v -x -a -m /tmp/uperf-test/uperfXML -i 1 | tee {{results.path}}/$(date +%s)-uperf"
+             seq {{uperf.rerun}} | xargs -I -- uperf -v -x -a -m /tmp/uperf-test/uperfXML -i 1 | tee {{results.path}}/$(date +%s)-uperf"
```
Comment (aakarshg) [inline roles/uperf-bench/templates/workload_store.yml.j2]: I don't like creating new results such that there's no concrete manner to link the reruns, but this seems to be out of scope of this PR. This should definitely be addressed when we retrospect on indexing results.

**Comment by aakarshg** [comment]
LGTM, merging it.
